test/framework/src/main/java/org/elasticsearch/test/hamcrest/ElasticsearchAssertions.java

```diff
@@ -636,7 +636,7 @@ public static void assertVersionSerializable(Version version, Streamable streama
          * a way that sucks less.
          */
         NamedWriteableRegistry registry;
-        if (ESIntegTestCase.isInternalCluster()) {
+        if (ESIntegTestCase.isInternalCluster() && ESIntegTestCase.internalCluster().size() > 0) {
```

Comment:
The assertion in `ESIntegTestCase.internalCluster().getInstance(NamedWriteableRegistry.class);` blows up if there are no nodes in the cluster. I stumbled on this while debugging